arch/arm/core/aarch64/reset.S

```diff
@@ -44,18 +49,65 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__reset)
 GTEXT(__start)
 SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 
-#ifdef CONFIG_SWITCH_TO_EL1
+	/* Setup vector table */
+	adr	x9, _vector_table
```

Comment:
Typo: I will change x9 to x19 either in another commit in this PR or update in next version.